### ISSUE TITLE
could you explain how to train full network step by step?

### ISSUE THREAD

**Issue by lerndeep** [kevinhkhsu/DA_detection#5]
for training domain adaptation from **KITTI to cityscape** dataset what step I need to flow step by step?
1) The first train faster runs with the KITTI dataset.
2) by using those train weights, train adaptation networks with cityscape datasets.
is it true?

**Comment by kevinhkhsu** [comment]
Before adaptation:
1. Train cycleGAN using [KITTI] and [Cityscapes]
2. Generate KITTI synthesized as Cityscapes images as the [synthetic domain]
3. Pre-train the detection network with [KITTI]
Domain adaptation steps:
1. Use the pre-trained detection network weights to train the adaptation network (source: [KITTI], target: [synthetic domain])
2. Finetune the adaptation network from the previous step (source: [synthetic domain], target: [Cityscapes])

Corresponding scripts and instructions to run the adaptation steps can be found in the README file under "Train your own model".

**Comment by lerndeep** [comment]
Thank you for your response.
I have one more question about labels of synthetic domain dataset. We need labels of synthetic domain dataset but how can I prepared labels because it will generate from cycleGAN?

**Comment by kevinhkhsu** [comment]
When generating the synthetic domain dataset, the images are actually translated from the source domain images (you can check the results, the object/scene does not change), so you can safely take the same labels to use as the ground truth.
